### DISCUSSION TITLE
practice-exam/practice-exam-17

### DISCUSSION THREAD

**Discussion by giscus** [kananinirav/AWS-Certified-Cloud-Practitioner-Notes#288] [Comments]
# practice-exam/practice-exam-17

If you are preparing for AWS Cloud Practitioner Exam, this guide will help you with quick revision before the exam. I share the notes I used to study and pass exam.

https://kananinirav.com/practice-exam/practice-exam-17.html

**Comment by Yussifmohammedibrahim** [comment]
please question 4 and 28 are B respectively

**Reply by tsid73** [reply]
yeah, you are correct

**Reply by ghost** [reply]
The AWS service that provides network connectivity in a hybrid architecture including the AWS Cloud is AWS Direct Connect.
Explanation:
AWS Direct Connect establishes a dedicated, private network connection from your on-premises data center, office, or colocation environment directly to AWS. This bypasses the public internet, offering increased bandwidth, reduced network costs, and a more consistent network experience, which are crucial for hybrid cloud deployments.
Amazon VPC (Virtual Private Cloud) allows you to provision a logically isolated section of the AWS Cloud where you can launch AWS resources in a virtual network that you define. While essential for organizing resources within AWS, it doesn't inherently provide the dedicated connectivity to on-premises environments required for a hybrid architecture.
AWS Directory Service is a managed service that provides directories for your AWS workloads, enabling you to integrate with existing on-premises directories or create new ones in the cloud. It's for identity and access management, not direct network connectivity.
Amazon API Gateway is a fully managed service that makes it easy for developers to create, publish, maintain, monitor, and secure APIs at any scale. While it can be used to expose resources in a hybrid environment, it's an API management service, not a dedicated network connectivity solution for linking on-premises networks to AWS at the infrastructure level.

**Reply by ghost** [reply]
The best service to use is B. AWS Simple Monthly Calculator, as it is designed to estimate the costs of running a new project on AWS. While the other options are useful for different tasks, the Simple Monthly Calculator is the primary tool for upfront cost estimation before you deploy.  
B. AWS Simple Monthly Calculator: This service is a planning tool for creating cost estimates for new or existing workloads on AWS. You can model your solutions, explore different instance types, and understand potential costs before you build. Note: The Simple Monthly Calculator is being retired, and you should use the AWS Pricing Calculator instead, which has the same core function of estimating costs for new projects. 
A. AWS TCO Calculator: This tool is used to compare the costs of running a workload on-premises versus on AWS, rather than just estimating the cost of the AWS project itself. 
C. AWS Cost Explorer API: The Cost Explorer API is for analyzing past and current costs and usage, not for estimating costs of a new, un-deployed project. 
D. AWS Budgets: This service is used to set custom budgets and receive alerts when costs or usage exceed your defined thresholds, but it doesn't help with the initial cost estimatio

**Comment by tiendat9825** [comment]
I believe the answer to question 22 is C

**Reply by tsid73** [reply]
no, linux ec2 are billed per second

**Reply by ghost** [reply]
An On-Demand Amazon Linux EC2 instance is billed per second, with a minimum charge of one minute (60 seconds). Since the instance ran for longer than the one-minute minimum, the customer will be billed for the exact duration of its runtime. 
Therefore, the customer will be billed for 3 hours, 5 minutes, and 6 seconds.
